### ISSUE TITLE
I can't move up to the speakers 

### ISSUE THREAD

**Issue by khaledkhoi** [grishka/Houseclub#188]
**Describe the bug**
I can't move up from the audience to the speakers

**To Reproduce**
1. Go to ' the person who raised his hand'
2. Click on 'invite him to speak '

**Expected behavior**
When one of the moderators wants me to speak they move me up from audience to the speakers 

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: [e.g.xiaomi note 8 pro]
 - OS: [e.g. Android 10]
 - Version [e.g. 12.0.6 ]

**Additional context**

**Comment by Awamy** [comment]
Just use any VPN or 1.1.1.1 from the Play store
the problem is with the egypt network not the app itself

**Comment by YoussefGoma** [comment]
> Just use any VPN or 1.1.1.1 from the Play store
> the problem is with the egypt network not the app itself

Yeah the problem has something to do with network restrictions in egypt but i think it can somehow be fixed as it works just fine with the ios version

**Comment by Error-404v2** [comment]
>  Just use any VPN or 1.1.1.1 from the Play store
> the problem is with the egypt network not the app itself

Can you explain further more what is that problem with Egypt network?

**Comment by Awamy** [comment]
> > Just use any VPN or 1.1.1.1 from the Play store
> > the problem is with the egypt network not the app itself
> 
> Can you explain further more what is that problem with Egypt network?

To be honest I don't know exactly what's the problem but there are some restrictions on the network in egypt that makes HouseClub face problems in egypt, You can't speak on the Android version, even IOS have some minor glitches.

all of these problems can be fixed using 1.1.1.1 on the Play Store/ App store

**Comment by YoussefGoma** [comment]
> > > Just use any VPN or 1.1.1.1 from the Play store
> > > the problem is with the egypt network not the app itself
> > 
> > 
> > Can you explain further more what is that problem with Egypt network?
> 
> To be honest I don't know exactly what's the problem but there are some restrictions on the network in egypt that makes HouseClub face problems in egypt, You can't speak on the Android version, even IOS have some minor glitches.
> 
> all of these problems can be fixed using 1.1.1.1 on the Play Store/ App store

I'm not talking technical but i think for some reason the request from the moderator for you to be a speaker never arrives to the client side it gets blocked for some reason. I think if the function dedicated to this part gets reviewed this problem can be solved

**Comment by Hesham942** [comment]
> > > > Just use any VPN or 1.1.1.1 from the Play store
> > > > the problem is with the egypt network not the app itself
> > > 
> > > 
> > > Can you explain further more what is that problem with Egypt network?
> > 
> > 
> > To be honest I don't know exactly what's the problem but there are some restrictions on the network in egypt that makes HouseClub face problems in egypt, You can't speak on the Android version, even IOS have some minor glitches.
> > all of these problems can be fixed using 1.1.1.1 on the Play Store/ App store
> 
> I'm not talking technical but i think for some reason the request from the moderator for you to be a speaker never arrives to the client side it gets blocked for some reason. I think if the function dedicated to this part gets reviewed this problem can be solved

Is it solved with you?

**Comment by khaledkhoi** [comment]
Yes thank you

**Comment by Mohammedsalah19** [comment]
> > > Just use any VPN or 1.1.1.1 from the Play store
> > > the problem is with the egypt network not the app itself
> > 
> > 
> > Can you explain further more what is that problem with Egypt network?
> 
> To be honest I don't know exactly what's the problem but there are some restrictions on the network in egypt that makes HouseClub face problems in egypt, You can't speak on the Android version, even IOS have some minor glitches.
> 
> all of these problems can be fixed using 1.1.1.1 on the Play Store/ App store

hello awamy is there version on play store?"
and i used secure VPN but still have same interface
